client/reader/discover/components/reddit/style.scss

```diff
@@ -2,12 +2,13 @@
 @import '@wordpress/base-styles/breakpoints';
 
 .discover-reddit {
+	padding: 0 10px;
 
 	&__instructions {
 		border: 1px solid #e2e2e2;
 		border-radius: 4px;
 		padding: 30px 15px;
-		margin: 100px auto 0;
+		margin: 50px auto 0;
```

Comment:
Currenlty on new tab we have "30px" space between search field and subscriptions so should we keep the spacing same?

Maybe it's better to remove these tab specific spacing and give `margin-bottom` to `.subscriptions-add-sites__form--container` which will make it consistent on both tabs.

Reply:
Done 👍